.github/actions/build-docker-images/action.yml

```diff
@@ -1,13 +1,23 @@
 name: Build docker images
 description: "Build docker images"
+inputs:
+  hash-agents:
+    description: "Build hash-agents image"
+    required: false
+    default: "false"
+  hash-graph:
+    description: "Build hash-graph image"
+    required: false
+    default: "false"
```

Comment:
Would this input be better as an array of images we want to build? Or does that give terrible error messages compared to the completion we could get from these bool options?

Reply:
An array with enum elements would be nice but I think this is not possible. With array of strings it’s easy to use the wrong string (e.g. `graph` or `hash_graph`)